xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -4732,7 +4733,6 @@ void CVideoPlayer::UpdatePlayState(double timeout)
     }
 
     state.canpause = m_pInputStream->CanPause();
-    state.canseek = m_pInputStream->CanSeek();
```

Comment:
Does this make pvr add-on API function CanSeek obsolete?

Reply:
yes. I did not drop the method because I wanted to avoid an API change

Reply:
Can you add a `//! @todo ` to the method declaration or definition?

Reply:
done

Reply:
I will take care of the api change for v19.